pom.xml

```diff
@@ -353,7 +353,7 @@
     <dependencies>
         <dependency>
             <groupId>net.sourceforge.plantuml</groupId>
-            <artifactId>plantuml-asl</artifactId>
+            <artifactId>plantuml</artifactId>
```

Comment:
Unfortunately, we can't use the GPL versioned plantuml from maven central.
See `lib/README.md` for details.